scripts/generate_landings_risk_report.py

```diff
@@ -847,6 +847,7 @@ def go(self, days: int) -> None:
         meta_bugs = self.get_meta_bugs(days)
 
         last_modified = db.last_modified(bugzilla.BUGS_DB)
+
```

Comment:
Please remove this unnecessary white line

Reply:
Done